### ISSUE TITLE
Make force Docker image public

### ISSUE THREAD

**Issue by pierrebeaucamp** [gravitational/force#91]
Force cannot be build on Darwin (#52), so the Docker image needs to be used. However, the one linked to in the docs (`docker pull gcr.io/kubeadm-167321/force:0.0.18`) is not public.

**Comment by webvictim** [comment]
You're correct. I can't make the image public without making all images in `kubeadm-167321` public (which I don't want to do) so we'll have to create a new project. I have done this, however we need to set up billing for the new project before we can use GCR (and I don't have billing permissions) so this is blocked on that.

In the meantime, try this image instead: `quay.io/gravitational/force:0.0.21`
